src/App/components/board/Board.tsx

```diff
@@ -1,30 +1,42 @@
-import React, { useCallback, useEffect, useState } from "react";
+import React, { useEffect, useState } from "react";
 import { Button } from "../../components";
-import { Player, Symbol } from "../../types";
-import { evaluateRows, evaluateColumns, evaluateDiagonals } from "./helpers";
+import { Player, Character } from "../../types";
+import { findWinner } from "./helpers";
 import "./Board.css";
 
 const CELLS = [0, 1, 2, 3, 4, 5, 6, 7, 8];
 
 function Block({
   index,
   board,
-  symbol,
+  character,
+  complete,
   onClick,
 }: {
   index: number;
-  board: Symbol[];
-  symbol: Symbol;
+  board: Character[];
+  character: Character;
+  complete: Boolean;
   onClick: (index: number) => void;
 }) {
+  let blockClass = "";
+
+  if (board[index] === "X") {
+    blockClass = "XClass";
+  } else if (board[index] === "O") {
+    blockClass = "OClass";
+  } else {
+    blockClass = `${character}Hover`;
+  }
+
+  if (complete) {
+    blockClass += " Complete";
```

Comment:
another common patter is using arrays to store the class names like so:

```ts
const classNames: string[] = [];
  if (board[index] === "X") {
    classNames.push("XClass");
  } else if (board[index] === "O") {
    classNames.push( "OClass");
  } else {
   classNames.push(`${character}Hover`);
  }

  if (complete) {
    classNames.push("Complete");
  }

 return (
    <div
      className={`Block ${classNames.join(' ')}`} />
);

```

Reply:
Awesome, that got rid of the space before "Complete" as well 🎉